saw-remote-api/python/saw/llvm.py

```diff
@@ -458,6 +476,16 @@ def field(base : SetupVal, field_name : str) -> 'FieldVal':
     """Returns a ``FieldVal`` using the field ``field_name`` of the struct ``base``."""
     return FieldVal(base, field_name)
 
+# It's tempting to name this `global` to mirror SAWScript's `llvm_global`,
+# but that would clash with the Python keyword `global`.
+def global_lvalue(name: str) -> 'GlobalLValueVal':
+    """Returns a ``GlobalLValueVal`` representing a pointer to the named global ``name``."""
+    return GlobalLValueVal(name)
+
+def global_initializer(name: str) -> 'GlobalInitializerVal':
+    """Returns a ``GlobalInitializerVal`` representing the value of the initializer of a named global ``name``."""
```

Comment:
See above discussion w.r.t. `global_lvalue`'s return type. I'm biased to prefer `SetupVal`.

Reply:
I've changed this to return `SetupVal`.